rero_ils/modules/acq_accounts/serializers.py

```diff
@@ -17,15 +17,38 @@
 
 """Acquisition account serialization."""
 
-from invenio_records_rest.serializers.response import search_responsify
+from invenio_records_rest.serializers.response import record_responsify, \
+    search_responsify
 
+from .api import AcqAccountsSearch
 from ..libraries.api import Library
 from ..serializers import JSONSerializer, RecordSchemaJSONV1
 
 
 class AcqAccountJSONSerializer(JSONSerializer):
     """Mixin serializing records as JSON."""
 
+    def preprocess_record(self, pid, record, links_factory=None, **kwargs):
+        """Prepare a record and persistent identifier for serialization."""
+        # Add some ES stored keys into response
+        es_hit = next(AcqAccountsSearch()
```

Comment:
If the search has no result `next` will raises an exception. Should we handle this?